common/djangoapps/util/views.py

```diff
@@ -383,6 +384,13 @@ def get_feedback_form_context(request):
 
     context["support_email"] = configuration_helpers.get_value('email_from_address', settings.DEFAULT_FROM_EMAIL)
 
+    #Update the tag info with 'enterprise_learner' if the user belongs to an enterprise customer.
+    enterprise_learner_data = enterprise_api.get_enterprise_learner_data(site=request.site, user=request.user)
+    if enterprise_learner_data:
+        enterprise_customer_name = enterprise_learner_data[0]['enterprise_customer']['name']
+        context["tags"]["learner_type"] = "enterprise_learner"
```

Comment:
I believe adding it to `tags` here will result in it also being sent to datadog.